annotate.py

```diff
@@ -245,14 +248,21 @@ def annotate_img(img_path, initials, size1=1, size2=1) :
 
 #Set up for directory of images with file structure for image segmentation
 #Function: annotate_dir
-#   input:
-#       img_dir & dataset - working directory. (directory named {img_dir}{dataset}).
-#       subset - Preffix for the two folders inside img_dir where images are located and labels are saved. Folders are subset_frames and subset_masks.
-#       size1 - size of the 1st brush circle, in px.
-#       size2 - size of the 2nd brush circle, in px.
-#       initials - First name and last name initials of the person doing the labelling. Recorded next to every PMT feature ID line in the text file.
-#       filename - The name of the .txt file that will be created.
-#Set up for directory of images with file structure for image segmentation
+# Inputs
+#    img_dir - image directory. Images are contained inside this folder.
+#    initials - First name and last name initials of the person doing the labelling. Recorded next to every PMT feature ID line in the text file.
+#    size1 - size of the 1st brush circle, in px.
+#    size2 - size of the 2nd brush circle, in px.
+# Outputs (for every image in {img_dir})
+#    {image_name}.png - A map of all the features in binary - features labelled in blue are recorded as RGB (1,1,1), and features in red are recorded as RGB (2,2,2).
```

Comment:
Seems repeated documentation above; can remove (since `annotate_img` is called below, developer should naturally go up and read that documentation)?

Reply:
oh n/m, i see you're describing the input/output of the function. can leave as is, or refer to `annotate_img` output (to minimize the amount of places you need to maintain if changing `annotate_img`)